bot/exts/holidays/halloween/spookygif.py

```diff
@@ -25,7 +25,7 @@ async def spookygif(self, ctx: commands.Context) -> None:
             # Make a GET request to the Giphy API to get a random halloween gif.
             async with self.bot.http_session.get(API_URL, params=params) as resp:
                 data = await resp.json()
-            url = data["data"]["image_url"]
+            url = data["data"]["images"]["original"]["url"]
```

Comment:
In the [giphy api docs](https://developers.giphy.com/docs/api/schema/#image-object) The `url` key on the `original` object isn't documented.

I am not sure if this is a documentation mistake on giphy's part, or if we're using an undocumented key that's likely to cause issues again in future.

To avoid all doubt, it might be better if we use `data["data"]["images"]["downsized"]["url"]`, as that's documented and also has the benefit of guaranteeing that the image is <2MB.

Reply:
Sounds logical. Change made.